cassandra/cqlengine/models.py

```diff
@@ -492,7 +494,10 @@ def column_family_name(cls, include_keyspace=True):
     def _raw_column_family_name(cls):
         if not cls._table_name:
             if cls.__table_name__:
-                cls._table_name = cls.__table_name__.lower()
+                if cls.__table_name_case_sensitive__:
+                    cls._table_name = cls.__table_name__
+                else:
+                    cls._table_name = cls.__table_name__.lower()
             else:
```

Comment:
IMO, I like the camelcase to '_' and lowercase transformation. This is exactly what I want to see by default. Django has the same behavior. Personnaly, I don't want to see camelcase in my db schema. It might be only me.

```
class Automobile(Model) -> automobile
class AutomobileGarage(Model) -> automobile_garage
```  

So, the issue here I think is only that specifying __table_name__ manually doesn't respect the case sensitivity, when it should.

Let's discuss this more tomorrow.